src/Form/Form.js

```diff
@@ -3,27 +3,54 @@ import classes from "./Form.module.css";
 import * as firebase from "../lib/firebase";
 
 const Form = props => {
+  const { setdbItems, dbItems, token, onEnterToken } = props;
```

Comment:
Fun fact about this, you can destructure this in line 5 itself.

```suggestion
  const Form = ({ setdbItems, dbItems, token, onEnterToken }) => {
```

Reply:
whaaaat? Awesome! Thanks!